services/galley/test/integration/API/Util.hs

```diff
@@ -2572,26 +2573,18 @@ mkProfile quid name =
 
 -- | Run the given action on a temporary galley instance with access to a mock
 -- federator.
---
--- The `resp :: FederatedRequest -> a` argument can be used to provide a fake
--- federator response (of an arbitrary JSON-serialisable type a) for every
--- expected request.
-withTempMockFederator ::
-  ToJSON a =>
-  (FederatedRequest -> a) ->
-  TestM b ->
-  TestM (b, [FederatedRequest])
-withTempMockFederator resp = withTempMockFederator' $ pure . encode . resp
```

Comment:
Given that `withTempMockFederator` is gone, how about renaming the remaining `withTempMockFederator'` to `withTempMockFederator`?

Reply:
It's not gone, it's the low-level function that is used by `withTempMockFederator'`. We could consider condensing all of these into a single function that takes a `Mock` parameter, but I would leave it for another PR.